internal/metrics/metrics.go

```diff
@@ -93,7 +82,7 @@ var (
 			Help:      "Count of total errors when forwarding RPC requests to upstreams.",
 		},
 		// jsonrpc_method is "batch" for batch requests
-		[]string{"chain_name", "client", "upstream_id", "url", "jsonrpc_method", "response_code", "jsonrpc_error_code"},
```

Comment:
Found that a couple of these `jsonrpc_error_code` fields were never populated with anything other than an empty string.  Better to just remove them.